agreement.md

```diff
@@ -0,0 +1,151 @@
+# Aragon Network Agreement
+
+## Section 0. Background
+The Aragon Network is a Decentralized Autonomous Organization that is building a dispute resolution protocol and other smart contract infrastructure.
+
+The Aragon Network is governed by Aragon Network Token (ANT) holders, who supported the initial creation of the network and under this Agreement will continue to guide its evolution through established governance processes.
+
+This Agreement exists to enumerate the shared mission, principles, and governance guidelines that bind ANT holders in their collaborative effort to create value for the Aragon Network.
+
+## Section 1. RFC 2119 compliance
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+## Section 2. Immutable guidelines
+The guidelines in this section SHALL NOT be amended. If any other guidelines in this Agreement conflict with these immutable guidelines, the immutable guidelines SHALL take priority for enforcement purposes. These guidelines apply to all proposals submitted to the Aragon Network organization.
+
+1. Proposals MUST be governed by this Agreement, or a future version of this Agreement as modified within the bounds of the Agreement.
+
+2. Disputes over a proposal relying on this Agreement MUST be resolved by Aragon Court.
+
+3. Proposals MUST NOT disproportionately benefit a majority of ANT holders voting on a proposal over the minority.
+
+4. Proposals MUST NOT attempt to cast a vote using ANT held in the address of any versions of the Agent app installed on the Aragon Network organization, or using ANT held in any Network contract that fulfills similar functionality as the Agent app.
```

Comment:
Clarifying question: what is the source of the ANT held in the Agent app? Is this ANT deposited/held by the various Aragon organizations/DAOs?

Reply:
Good question. It could come from anywhere but the main source I could see it coming from would be newly minted ANT that is not immediately spent. Another source could be ANT that was allocated for a particular purpose but was returned to the DAO for some reason (e.g. a milestone was not met on a grant, the grant recipient team disbanded, etc).

Reply:
Ah... so this is more like a network treasury?

Reply:
Yes, the Agent is/ can be used as a replacement for the Vault app, and holds all of the ETH/ERC-20/ERC-721 tokens belonging to the Network.